string data # the request
---
bool success # the response